portfolio/src/main/webapp/index.html

```diff
@@ -119,27 +119,20 @@ <h2>Dominique's Portfolio</h2>
           </ul>  
       </nav>
 
-      <article>
-          <h3> All About Me!</h3>
+      <main>
+          <h2> About Me</h2>
           <p> Hi, my name is Dominique Buford. I am from Dallas, Texas, I'm a sophomore at Duke University, and I plan
```

Comment:
Text here will get hidden if I scroll to the top https://screenshot.googleplex.com/LXpSYGf1Jy2

Reply:
Are you able to `resolve conversation` from your side? Feel free to close any comments that are fixed.